svgdigitizer/electrochemistry/cv.py

```diff
@@ -735,10 +736,68 @@ def comment(self):
 
         return comments[0].value
 
+    @property
+    def simultaneous_measurements(self):
+        r"""
+        Returns the names of additional measurments which are plotted
```

Comment:
```suggestion
        The names of additional measurments which are plotted
```
Since this is a property, it does not perform an action.